waku/node/v2/waku_types.nim

```diff
@@ -204,14 +212,17 @@ proc generateRequestId*(rng: ref BrHmacDrbgContext): string =
   brHmacDrbgGenerate(rng[], bytes)
   toHex(bytes)
 
-proc computeIndex*(msg: WakuMessage): Index =
-  ## Takes a WakuMessage and returns its Index 
+proc id*(msg: WakuMessage): MDigest[256] =
+  ## Generates an ID from the message
   var ctx: sha256
   ctx.init()
   ctx.update(msg.contentTopic.toBytes()) # converts the contentTopic to bytes
   ctx.update(msg.payload)
-  let digest = ctx.finish() # computes the hash
+  result = ctx.finish() # computes the hash
   ctx.clear()
 
-  result.digest = digest
+proc computeIndex*(msg: WakuMessage): Index =
```

Comment:
Oh computeIndex is here, diff was confusing me

Reply:
I moved it back, realized I can just use computeIndex.